config/settings.yml

```diff
@@ -80,6 +80,8 @@
   :scan_via_host: true
   :use_vim_broker: true
   :use_vim_broker_ems: true
+:container_logging:
+  :common_logging_route: logging-kibana-ops
```

Comment:
> This feels wrong as a setting in the general settings...feels like it should a per-provider settings or perhaps a special endpoint on each provider.
 
@Fryguy Like Federico commented, We are setting this route in the default installation of Openshift in openshift-ansible so it is safe to assume this is a default. per-provider settings (as custom attributes? ) are also not very easy to set by the user, and this is a feature that we hope that will be widely used.
Maybe we can leave this as a "default" setting for all the providers and also allow for a per-provider setting via custom attributes ?